src/python/gennylib/genny_auto_tasks.py

```diff
@@ -263,7 +263,6 @@ def construct_all_tasks_json():
     :return: json representation of tasks for all workloads in the /src/workloads directory relative to the genny root.
     """
     c = Configuration()
-    c.exec_timeout(64800)  # 18 hours
```

Comment:
Did you get a chance to file the ticket to fix shrub.py?

Reply:
Oof spent some time with a local fix before deciding to just use a command, but I'll do that now.